.github/workflows/linting.yml

```diff
@@ -28,6 +30,14 @@ jobs:
         with:
           go-version-file: 'go.mod'
           go-package: ./...
+      - name: Comment on PR
+        if: steps.govulncheck.outcome == 'failure'
+        uses: thollander/actions-comment-pull-request@fabd468d3a1a0b97feee5f6b9e499eab0dd903f6 # v2.5.0
```

Comment:
Is it difficult if we use the gh api instead of another action?

Reply:
There is an approach that uses curl and the API and one that uses the script action to allow calling gh-action functions. I just briefly looked into it though. I suspect they'd also need write rights in order to post the comment.

Reply:
I was actually referring to the `gh` client as it's already available in the action but I haven't played around with it that much to tell if it will actually work properly for our use case (comment once -> edit afterwards)

Reply:
Looks like that might also be feasible but I actually discovered an even better way using a rather underdocumented feature: The env var `$GITHUB_STEP_SUMMARY` which you can just echo into and it displays this as a comment on the discussions tab. I'll try this out and update this draft.

Reply:
Nevermind, it displays on the summary and not discussion tab. I added a way that uses the gh client as you noted now.